.gitignore

```diff
@@ -2,3 +2,4 @@ NOTES.md
 drone-local-cache
 drone-volume-cache
 .env
+.idea/
```

Comment:
This is developer dependent, please drop that and put that into a local global gitignore